shared/actions/platform-specific/index.native.tsx

```diff
@@ -241,20 +241,29 @@ function* persistRoute(state, action: ConfigGen.PersistRoutePayload) {
   )
 }
 
+const updateMobileNetState = (type: string) => {
```

Comment:
this should just be a chainAction of `osNetworkStatusChanged`

Reply:
we prefer side effects to be chained based on actions vs having to explicitly plumb through calls